### ISSUE TITLE
amd hashrate n/a

### ISSUE THREAD

**Issue by pmrk74** [xmrig/xmrig#3148]
I want to mine Monero currency through this program, but the problem is that the program shows the hash rate n/a, it also shows this error:
opencl disabled (failed to start thread)

how i can fix this?

my system;
windows 11
amd 580 8gb

**Comment by SChernykh** [comment]
First of all, you have to mine with CPU, not GPU. GPUs are very slow on RandomX. Second, you need to show the screenshot of XMRig window when it starts and shows this error, or I can't tell you what exactly went wrong.

**Comment by pmrk74** [comment]
https://s6.uupload.ir/files/untitled-ss_9a1o.png

**Comment by Lonnegan** [comment]
Why do you say "amd hashrate n/a"?! You have an Intel CPU!

**Comment by SChernykh** [comment]
If you really want to mine on your AMD GPU, try Windows 10 and probably older AMD drivers. But hashrate will be very low, it's better to mine Monero on CPU only and use GPUs for something else.

**Comment by pmrk74** [comment]
yes i have intel cpu but i want to do this with gpu because cpu power is low for this task

**Comment by SChernykh** [comment]
Radeon RX 580 will give you only 470 h/s even if you manage to make it work.

**Comment by pmrk74** [comment]
So, how to solve the hashrate n/a problem?

**Comment by SChernykh** [comment]
- Mine on CPU
- Windows 10 and older driver to mine on GPU

**Comment by Spudz76** [comment]
Guess at which drivers aren't buggy:
* “good” AMD drivers for Windows: from 18.12.1.1 to 19.7.5 (inclusive), and from 19.12.2 to 20.11.1 (inclusive)
